lib/mayaUsd/ufe/UsdRotatePivotTranslateUndoableCommand.h

```diff
@@ -54,14 +54,10 @@ class MAYAUSD_CORE_PUBLIC UsdRotatePivotTranslateUndoableCommand : public Ufe::T
 	bool translate(double x, double y, double z) override;
 
 private:
-	void conditionalCreateItem();
-
-private:
+	Ufe::Path fPath;
 	UsdSceneItem::Ptr fItem{nullptr};
-	UsdPrim fPrim;
 	UsdAttribute fPivotAttrib;
 	GfVec3f fPrevPivotValue;
-	Ufe::Path fPath;
```

Comment:
@ppt-adsk  remove unused variables that I missed to clean up in 2 previous commits.